src/osvr/Macros.cmake

```diff
@@ -34,10 +39,19 @@ macro(osvr_add_library)
 
     set_target_properties(${LIBNAME_FULL} PROPERTIES
         FOLDER "${LIB_FOLDER}")
-
-    generate_export_header(${LIBNAME_FULL}
-        BASE_NAME ${EXPORT_BASENAME}
-        EXPORT_FILE_NAME Export.h)
+        
+    # Are we building or using this library?
+    get_target_property(EXPORT_IMPORT_CONDITION ${LIBNAME_FULL} DEFINE_SYMBOL)   
```

Comment:
I don't follow the purpose of this section of code.  CMake auto-defines some preprocessor define `${LIBNAME_FULL}_EXPORTS`